modules/rpc/thriftrw-plugin-thriftsync/README.md

```diff
@@ -10,146 +10,150 @@ Following examples show how thriftsync syncs handler code with the updated thrif
 
 ```thrift
 service TestService {
-    string testFunction(1: string param)
+  string testFunction(1: string param)
 }
 ```
 
 ```go
 package main
 
 import (
-	"go.uber.org/fx"
-	"testservice/testservice/testserviceserver"
-	"go.uber.org/fx/service"
-	"go.uber.org/yarpc/api/transport"
+  "context"
+
+  "testservice/testservice/testserviceserver"
+  "go.uber.org/fx/service"
+  "go.uber.org/yarpc/api/transport"
 )
 
 type YARPCHandler struct {
-	// TODO: modify the TestService handler with your suitable structure
+  // TODO: modify the TestService handler with your suitable structure
 }
 
 // NewYARPCThriftHandler for your service
 func NewYARPCThriftHandler(service.Host) ([]transport.Procedure, error) {
-	handler := &YARPCHandler{}
-	return testserviceserver.New(handler), nil
+  handler := &YARPCHandler{}
+  return testserviceserver.New(handler), nil
 }
 
-func (h *YARPCHandler) TestFunction(ctx fx.Context, param *string) (string, error) {
-	// TODO: write your code here
-	return "", nil
+func (h *YARPCHandler) TestFunction(ctx context.Context, param *string) (string, error) {
+  // TODO: write your code here
+  panic("To be implemented")
 }
 ```
 **New function added to thrift file**
 
 ```thrift
 service TestService {
-    string testFunction(1: string param)
+  string testFunction(1: string param)
 
-    string newtestFunction(1: string param)
+  string newtestFunction(1: string param)
 }
 ```
 
 ```go
 package main
 
 import (
-	"go.uber.org/fx"
-	"testservice/testservice/testserviceserver"
-	"go.uber.org/fx/service"
-	"go.uber.org/yarpc/api/transport"
+  "context"
+
+  "testservice/testservice/testserviceserver"
```

Comment:
style: this is a "project" import so should be separate from 3rd-party (go.uber.org) imports